staging/src/k8s.io/component-base/cli/globalflag/globalflags.go

```diff
@@ -66,3 +66,19 @@ func Register(local *pflag.FlagSet, globalName string) {
 		panic(fmt.Sprintf("failed to find flag in global flagset (flag): %s", globalName))
 	}
 }
+
+// pflagRegister adds a flag to local that targets the Value associated with the Flag named globalName in global
+func pflagRegister(global, local *pflag.FlagSet, globalName string) {
```

Comment:
these are private now. Intentional?

Reply:
yes, just move from cmd/kubelet/app/options/globalflags.go file in this commit log. In next commit log, we change it into public